docs/development/contribution-documentation/documentation-process/README.md

```diff
@@ -137,7 +155,36 @@ In the field "Reviewers" select "opf/doc-writers".
 
 ![select reviewer for documentation](select-reviewer-for-documentation.png)
 
-## Step 15: Wait for feedback from the reviewers
+## Step 17: Wait for feedback from the reviewers
 
 ... and hopefully it is all LGTM which means "Looks good to me(rge). Congrats to your first contribution to the OpenProject documentation. We appreciate your effort :-)
 
+
+
+## Appendix A: Rebase your fork when the release branch changes
+
+*(e.g. from `release/12.0` to `release 12.1`)*
```

Comment:
I'm missing here why we are doing this. Is this just to get the new release branch into our fork?

If we chose "contribute to the parent project" after cloning the repository then this would be simply a matter of choosing the branch in the upstream repository in github desktop (i.e. click on upstream/release/12.1 in the select box):

![image](https://user-images.githubusercontent.com/158871/174754034-4a9a0cbd-bed0-46ba-bb61-1827dfe0ea5a.png)

This will then create the release/12.1 branch in your local repository which then can also just be pushed to the fork (origin). This is more intuitive and less error-prone IMHO.
Accordingly, I suggest to stick with "contribute to the parent project" in the repo settings.

After the new release branch has been pushed it would make sense to update the fork's default branch again.
So this section in the appendix could be called "Updating the release branch" instead.